src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/Example1.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="EqualsHashCode"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
+// xdoc section -- start
+class Example1 {
+  public int hashCode() { // violation, no valid 'equals'
+    return 0;
+  }
+  public boolean equals(String o) { return false; }
+}
+class ExampleNoHashCode {
```

Comment:
please add empty line before each class, 
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/0e05b8f_2024064831/checks/coding/equalshashcode.html#Example1-code is hard to read without spaces.